src/metabase/driver.clj

```diff
@@ -948,11 +952,21 @@
 
   - [:bigint]
   - [[:varchar 255]]
-  - [:generated-always :as :identity :primary-key]"
+  - [:generated-always :as :identity]"
   {:changelog-test/ignore true, :added "0.47.0", :arglists '([driver upload-type])}
   dispatch-on-initialized-driver
   :hierarchy #'hierarchy)
 
+(defmulti create-auto-pk-with-append-csv?
```

Comment:
this looks like something we should use driver feature for?

Reply:
What's the benefit you see? I felt like a multimethod is better because it (a) is more hidden away away it doesn't complicate the feature list, and (b) it doesn't require the database instance to be in scope in the caller, unlike database-supports?